README.md

```diff
@@ -13,28 +13,40 @@ It is assumed a snapshot of the CSMP agent's source repository has been acquired
 
 Change your default directory to the root of the CSMP Agent directory structure.
 
-1. Install build tools.
+### Install build tools.
 Install build-essential package (gcc compiler, make, etc.) as described here ... https://help.ubuntu.com/community/InstallingCompilers
 
-2. Confirm build target platform
+### Confirm build target platform
 If you are going to build for a different target platform, please set the correct gcc compiler for your target platform by modifying the line "CC=gcc" in "Makefile".
 
-3. Build
+### Build
 >   chmod 777 build.sh
-- Linux
+
+#### Linux
 >  ./build.sh linux
 
-- FreeRTOS
+#### FreeRTOS
 > git submodule update --init --recursive
 > ./build.sh freertos
 
 If everything goes well, you should see "CsmpAgentLib_sample" executable in "sample" directory.
 
-4. clean
+#### Silicon Labs - Wi-SUN EFR32 Platforms
+
+Silicon Labs Wi-SUN EFR32 platforms support the CSMP Agent with the Simplicity SDK version 2024.6.0. 
+
+The demonstration requires using a Silicon Labs Wi-SUN Linux Border Router plus the Silicon Labs Wi-SUN node that will connect to Cisco FND.  
```

Comment:
"Silicon Labs Wi-SUN Linux Border Router" does it ?

Reply:
yes, the demonstration documented in the Readme under Vendors/Silabs requires using our Linux border router.